tools/create-app.bash

```diff
@@ -5,10 +5,23 @@
 
 # Params: $1: output directory
 
-# Sign app for release
+### Assets ###
+
+# Place logo in the assets file
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+mv ./logo-hippie.png $dir_path/../app/assets/images/
```

Comment:
I think this logo image should not be hardcoded that way. The script precondition says the logo name should be `logo.png`, and it must be searched by the script in an images directory, coming from an argument. In line 13 we assume the logo is always at the root dir (where the script is placed).